src/test/java/seedu/budgetbuddy/SavingListTest.java

```diff
@@ -2,14 +2,19 @@
 
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
-
 import seedu.budgetbuddy.exception.BudgetBuddyException;
-
+import java.util.logging.Level;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.logging.Logger;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 public class SavingListTest {
 
+    private static final Logger LOGGER = Logger.getLogger(SavingListTest.class.getName());
```

Comment:
Good use of the Logger. Should we use a GLOBAL_LOGGER_NAME instead to easily configure logger options?